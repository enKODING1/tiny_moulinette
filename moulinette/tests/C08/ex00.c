#include <stdio.h>
#include "ft.h"

int main(void)
{
    int a = 4;
    int b = 2;
    ft_putchar('a');
    ft_putstr("Hi");
    ft_strcmp("Hello", "World");
    ft_strlen("Hello 42 Gyeongsan");
    ft_swap(&a, &b);
    return 0;
}

void ft_putchar(char c)
{
    printf("ft_putchar called: %c\n", c);
}

void ft_putstr(char *str)
{
    printf("ft_putstr called: %s\n", str);
}

int ft_strcmp(char *s1, char *s2)
{
    printf("ft_strcmp called:\n\t%s\n\t%s\n", s1, s2);
    return 0;
}

int ft_strlen(char *str)
{
    printf("ft_strlen called: %s\n", str);
    return 0;
}

void ft_swap(int *a, int *b)
{
    printf("ft_swap called: %d %d\n", *a, *b);
}
