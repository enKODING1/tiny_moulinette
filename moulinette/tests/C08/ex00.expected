ft_putchar called: a
ft_putstr called: Hi
ft_strcmp called:
	Hello
	World
ft_strlen called: Hello 42 Gyeongsan
ft_swap called: 4 2
